splunk_eventgen/lib/concurrent/futures/__init__.py

```diff
@@ -1,23 +0,0 @@
-# Copyright 2009 Brian Quinlan. All Rights Reserved.
```

Comment:
`concurrent.futures` is a third party lib that should not be formatted.  And best practice is not checked into the source code.